go.mod

```diff
@@ -5,3 +5,5 @@ require (
 	github.com/kr/pretty v0.1.0 // indirect
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127
 )
+
```

Comment:
Minor: given that this is a public SDK, it would be better if any Go version change was in a separate PR, instead of here.